cloudigrade/account/serializers.py

```diff
@@ -129,6 +134,27 @@ def create(self, validated_data):
             )
         return account
 
+    def add_openshift_tag(self, session, ami_id, ami_region, image):
```

Comment:
Not a blocker, but possibly rename this function since we aren't always adding the openshift tag? Possibly `check_for_openshift` or something along those lines?